### DISCUSSION TITLE
Permanent SMTP delivery error to custom domains

### DISCUSSION THREAD

**Discussion by lmaccount** [postalserver/postal#1830] [Help with using Postal]
We have a self-hosted postal server and certain email addresses are getting Permanent SMTP delivery error with every attempt to send an email, resulting in a hard fail. It appears to only effect email addresses with custom domain names.
**"Permanent SMTP delivery error when sending to _[CUSTOM EMAIL DOMAIN NAME]_-co-nz.mail.protection.outlook.com (104.47.71.138)"**
Where the IP address is Microsoft's, mail-sy4aus010138.inbound.protection.outlook.com
However emails are successfully sent to @outlook.com email addresses, so the issue does not appear to be with Outlook itself.

The technical details say **"550 5.7.511 Access denied, banned sender _[IP ADDRESS]_. To request removal from this list please forward this message to delist@messaging.microsoft.com. For more information please go to http://go.microsoft.com/fwlink/?LinkId=526653. AS(1410) [ME3AUS01FT016.eop-AUS01.prod.protection.outlook.com]"**
We have tried forwarding these error messages and checked the provided links but all attempts for support from Outlook haven't solved the issue. We contacted Outlook to try whitelist our IP but there were no blocks on it. 

Has anyone experienced similar issues and managed to find a solution?

**Comment by willpower232** [comment]
This seems to be an office 365 specific issue. It may be that the company in question has stringent spam rules that they control but Microsoft do not. You may need to contact the recipient directly to resolve.
